profiling/src/profiling.rs

```diff
@@ -183,7 +183,7 @@ unsafe fn extract_function_name(func: &zend_function) -> String {
      * erring on the side of caution and returning early.
      */
     if method_name.is_empty() {
-        return String::new();
+        return None;
```

Comment:
praise: I like this API change very much 👍

Reply:
Thank you :)